recipes/graphviz/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set version = "2.38.0" %}
+{% set win_version = version[:-2] %}
+
+package:
+  name: graphviz
+  version: {{ version }}
+
+source:
+  fn:  graphviz-{{ version }}.tar.gz  # [unix]
+  url: http://www.graphviz.org/pub/graphviz/stable/SOURCES/graphviz-{{ version }}.tar.gz  # [unix]
+  md5: 5b6a829b2ac94efcd5fa3c223ed6d3ae  # [unix]
+
+  fn:  graphviz-{{ win_version }}.zip  # [win]
+  url: http://www.graphviz.org/pub/graphviz/stable/windows/graphviz-{{ win_version }}.zip  # [win]
+  md5: d028f49e047e2a25fdb7c0aa323a3945  # [win]
```

Comment:
There wouldn't be a separate Windows section. Hope that is ok. I don't remember whether there was a significant difference between the two.